lib/compress.js

```diff
@@ -4652,18 +4652,20 @@ merge(Compressor.prototype, {
             });
         }
         // x ? y(a) : y(b) --> y(x ? a : b)
+        var arg_index;
         if (consequent instanceof AST_Call
             && alternative.TYPE === consequent.TYPE
-            && consequent.args.length == 1
-            && alternative.args.length == 1
```

Comment:
Please add this bailout here to reduce CPU:
```
            && consequent.args.length == alternative.args.length
```

Reply:
That makes sense - thanks!